css-pseudo-4/Overview.bs

```diff
@@ -1147,6 +1147,20 @@ Tree-Abiding Pseudo-elements</h2>
   non-inheritable properties take their <a>initial values</a> as usual.
   [[CSS-CASCADE-4]]
 
+<h3 id=partlike>
+Part-Like Pseudo-Elements</h3>
+
+  A subset of [=tree-abiding pseudo-elements=],
+  the <dfn>part-like pseudo-elements</dfn>,
```

Comment:
Not sure I like this name. Can we pick something that isn't a reference to `::part()`?

Reply:
Yup, would love a different name, but having a problem coming up with one. Problem is coming up with something that's not *just* a synonym of "abiding". We could try to skirt a little farther in the "Tree-X" namespace - tree-located? tree-positioned? tree-structural?- to indicate that not only do they abide by the tree structure *generally* (insofar as having a parent/child relationship) but actually have a definite *position* in that tree (and thus know their sibling relationships too).

Or we could go further afield. Semi-real? Pseudo-real?

Reply:
"Tree-structural" is actually really grabbing me, given its parallel with the tree-structural pseudo-classes.

Reply:
Well `::marker` has a definite position in the tree, but you're not including it in this set of things afaict. So it's not about it's tree position, it's about the fact that generates a box that behaves exactly like any other element's generated box.

Reply:
I was under the impression that the *reason* ::marker didn't act like ::before/etc is *because* we specifically hadn't nailed down its exact tree position. If it *does* have an exact position, then *what* is the reasoning behind limiting it? (Besides applying a few one-offs, like disallowing it from being `display: list-item` or something; that's fine for this new category of pseudos to do.)